### PR TITLE
Add option to hide the sidebar menu when you need.

### LINKED ISSUE #836: How to disable the right side toolbar?
On my project I want a very simple Rich Text Editor and I saw that I can configure which options of slate I want. 
My question is? How can I disable the right sidebar menu? Should I just hide with CSS or there is a more elegant way that the lib allows me to do?

### PR DESCRIPTION
**Pull request by matheusrocha89**
## Proposed changes
Sometimes you just want a simple editor without so many options, this PR allows you to hide the sidebar menu.

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Closing issues

Fixes #836

### CODE REVIEW

File: packages/ui/src/EditorUI/__tests__/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    expect(component).toMatchSnapshot();
+  });
+
+  it('renders correctly with option to hide the editor sidebar', () => {
+    const component = shallow(<EditorUI {...defaultProps} hideEditorSidebar />);
+
+    expect(component.find('DisplayModeToggle')).toHaveLength(0);
+  });
+});
```
Comment (macrozone) [inline packages/ui/src/EditorUI/__tests__/index.test.tsx:27]: nice! we should definitly start to do more component tests

**Comment by macrozone** [comment]
perfect!

**Comment by macrozone** [comment]
@matheusrocha89 can you rename the commit message to:

`feat: Add option to hide the sidebar menu when you don't need it`


the `feat:` part is important, because that will trigger the version bump and the automatic release

**Comment by matheusrocha89** [comment]
Sure, no problem.

File: packages/ui/src/EditorUI/__tests__/index.test.tsx (hunk 1 above)
Comment (matheusrocha89) [reply packages/ui/src/EditorUI/__tests__/index.test.tsx:27]: There are a lot of approaches for tests in React. There are people that prefer to test on a big scale. Usually, I do in a component scale, but you can use the best one for your project.

**Pull request merged**